cloudconfig.tpl

```diff
@@ -836,11 +836,69 @@ write_files:
         sudo touch $cronTaskPath
         sudo chmod 777 $cronTaskPath
 
-        echo "0 0 * * * /home/wowza/renew-cert.sh
-        " > $cronTaskPath
+        echo "0 0 * * * /home/wowza/renew-cert.sh" >> $cronTaskPath
+        
+        if [[ $HOSTNAME == *"prod"* ]] || [[ $HOSTNAME == *"stg"* ]]; then
+           echo "10 0 * * * /home/wowza/check-cert.sh" >> $cronTaskPath
+        fi
 
         sudo -u wowza bash -c "crontab $cronTaskPath"
+  - owner: wowza:wowza
+    permissions: 0775
+    path: /home/wowza/check-cert.sh
+    content: |
+        #!/bin/bash
+
+        # Project
+        project="${project}" 
```

Comment:
project will always be CVP so do we need to pass this in?

Reply:
true, was just templated for VH and CVP